### ISSUE TITLE
torch.nn.AvgPool2d() not work right

### ISSUE THREAD

**Issue by LaiPiXiong** [pytorch/pytorch#6494]
In the Doc, the Wout from torch.nn.AvgPool2d() is 
![image](https://user-images.githubusercontent.com/15033269/38591738-cc09aed0-3d6a-11e8-9fcc-7bf138789bae.png)
But, when input is 512 * 7 *7,  pool = torch.nn.AvgPool2d(kerel_size = 3, padding = 1), it's out is 512*3*3, which should be 512*7*7

**Comment by SsnL** [comment]
stride implicitly equals to kernel_size. It's in the doc.

**Comment by LaiPiXiong** [comment]
Thanks.
